redaxo/src/core/lang/de_de.lang

```diff
@@ -167,6 +167,7 @@ setup_413 = Zeitzone ist ungültig!
 setup_414 = Schritt 4 erneut durchführen
 setup_415 = Folgender Fehler wurde gemeldet: {0}
 setup_416 = REDAXO-Einstellungen
+setup_417 = Datenbank ok
```

Comment:
Eigentlich hatten wir folgende Regeln vereinbart:
- Keine neuen Keys nur für die Console
- In der Console bevorzugt `rex_i18n` gar nicht nutzen, sondern alle Messages direkt schreiben
- Nur wenn Funktionen nun mal eine Result-Message oder so liefern, dann kann die natürlich genutzt (und dekodiert) werden

Können es auch noch mal umdefinieren, aber hätte es ganz gerne einheitlich.

Reply:
stimmt, hatte ich vergessen